app/code/Magento/Theme/view/frontend/templates/html/topmenu/renderer.phtml

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
@sshymko Agreed. "The .phtml still needs work as I have just copied that directly from the .php file for now. Happy to continue and attempt to redo that if this pull request is of use, but mabey that is better done by a frontend dev internally.".